lib/charms/ovn_charm.py

```diff
@@ -555,12 +649,12 @@ def custom_assess_status_last_check(self):
         :rtype: (status, message) or (None, None)
         """
 
-        if not self.valid_config:
-            message = ('Wrong format for bridge-interface-mappings. '
-                       'Expected format is space-delimited list of '
-                       'key-value pairs. Ex: "br-internet:00:00:5e:00:00:42 '
-                       'br-provider:enp3s0f0"')
-            return 'blocked', message
+        if self.options.validation_errors:
```

Comment:
Now that this method checks multiple things an update of the docstring is also in order